src/components/atoms/ThresholdChart.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+
+import { scaleLinear, scaleTime } from '@visx/scale';
+import { GridRows, GridColumns } from '@visx/grid';
+import { Group } from '@visx/group';
+import { AxisLeft, AxisBottom } from '@visx/axis';
+import { curveBasis } from '@visx/curve';
+import { LinePath } from '@visx/shape';
+import { Threshold } from '@visx/threshold';
+
+interface IData {
+    time: string;
+    value: number;
+}
+
+export type ThresholdChartProps = {
```

Comment:
You should probably decide on whether you want to use interfaces or types to avoid confusion.

Paging @IT2901-SINTEF01/frontend.

Reply:
Can add that interfaces can only represent objects, whereas types can represent anything.

Reply:
Ait, any preferences?

Reply:
If you wish you can also use `type` for non-objects and `interface` for objects.

Reply:
Do I still name the type starting with capital I?

Reply:
Not if you use `type`. `I`-prefix is only for `interface`s.